src/org/usfirst/frc2175/subsystem/drivetrain/DrivetrainSubsystem.java

```diff
@@ -13,12 +13,14 @@
 import edu.wpi.first.wpilibj.PIDSourceType;
 import edu.wpi.first.wpilibj.RobotDrive;
 import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 
 public class DrivetrainSubsystem extends BaseSubsystem {
     private Talon leftDriveTalon;
     private Talon rightDriveTalon;
     private Encoder leftDriveEncoder;
     private Encoder rightDriveEncoder;
+    private Gyro gyro;
```

Comment:
Ok, I can't leave a comment below, but take a look lower in the file, around lines 35-38. You need to actually set this variable to something, otherwise the gyro methods will freak out when we try to run them.

So like how you have `leftDriveEncoder = wiringConfig.getLeftDriveEncoder();` down below, you need to also put in `gyro = wiringConfig.getGyro();`.